packages/sync/src/provider.ts

```diff
@@ -139,7 +169,7 @@ export class SyncProvider {
 					Y.applyUpdate( ydoc, Y.encodeStateAsUpdate( initialDoc ) );
 				},
 				LOCAL_SYNC_PROVIDER_ORIGIN,
-				false
+				true
```

Comment:
This change to the `local` parameter is intentional. Previously, it had no effect because we ignored it and checked the origin against a list of known local origins. Now we check `transaction.local` so it needs to be updated to properly reflect a local update.

Reply:
Would add this as a comment here, and or at the top of this class to make sure it's known how important it is that this is followed.